base.py

```diff
@@ -426,14 +467,41 @@ class MBDynSettingsObject(bpy.types.PropertyGroup):
             update = update_label
             )
 
-    # String representing MBDyn's node string label assigned to the object. 
+    # String representing MBDyn's node string label assigned to the object.
     # Non-"not assigned" only if a .lab file with correct syntax is found
     string_label = StringProperty(
             name = "MBDyn's node or joint string label",
             description = "String label of MBDyn's node assigned to the object (if present)",
             default = "not assigned"
             )
 
+    #String representing path of the output directory
```

Comment:
My mistake. I did that long ago when I didn't really understand the significance of SettingsObject and SettingsScene. Changing it now.
And as you reported, this is what causes the above error you reported in an empty scene, as there are no objects to hold those properties.